### PR TITLE
sig-contribex: annual report 2023

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
Adding SIG Contributor Experience annual report for 2023

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/community/issues/7770

cc: @kubernetes/sig-contributor-experience-leads

### CODE REVIEW

File: sig-contributor-experience/annual-report-2023.md [hunk 1]
```diff
@@ -4,6 +4,43 @@
...
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - got access to Linkedin
+        - started process to get access to kubernetes twitter account
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
+        - SIG spotlight blogs
+            - how many?
+            - how many by new contributors?
+        - upates to leadership roles?
```
Comment (Priyankasaggu11929) [inline sig-contributor-experience/annual-report-2023.md:38]: @kaslin, is it possible to fetch these numbers for SIG spotlight blogs?

File: sig-contributor-experience/annual-report-2023.md [hunk 2]
```diff
@@ -3,17 +3,60 @@
...
--->
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+
+    - Mentoring subproject needs a dedicated leadership, responsible for running both (i) group mentoring cohorts for Kubernetes SIGs & WGs, (ii) external mentoring programs (LFX Mentorship, Google Summer of Code, Google Summer of Docs, Outreachy, etc)
+
+
+    - Elections subproject is need of a lead, who will own & maintain election software, pick election officers, and  help groups in the Kubernetes project, who want to use the election software.
+
```
Comment (Priyankasaggu11929) [inline sig-contributor-experience/annual-report-2023.md:59]: Follow up from the `April 21, 2024` ContribEx bi-weekly meeting:

Hello @jberkus, ping for collecting specifics for the help needed for mentoring & elections subproject. Thanks!

File: sig-contributor-experience/annual-report-2023.md [hunk 3]
```diff
@@ -3,17 +3,61 @@
...
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - got access to Linkedin
+        - started process to get access to kubernetes twitter account
```
Comment (mfahlandt) [inline sig-contributor-experience/annual-report-2023.md:41]:
```suggestion
        - got acces to kubernetesio twitter account
```

**Comment by palnabarun** [comment]
/assign @mrbobbytables 
(steering liaison)

File: sig-contributor-experience/annual-report-2023.md [hunk 4]
```diff
@@ -3,17 +3,61 @@
...
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
+        - SIG spotlight blogs
+            - how many?
+            - how many by new contributors?
+        - new subproject leadership roles?
```
Comment (palnabarun) [inline sig-contributor-experience/annual-report-2023.md:46]: cc: @kaslin

File: sig-contributor-experience/annual-report-2023.md [hunk 5]
```diff
@@ -54,12 +105,12 @@
 ## Operational
 
 Operational tasks in [sig-governance.md]:
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [X] [README.md] reviewed for accuracy and updated if needed
+- [X] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
 - [ ] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
```
Comment (palnabarun) [inline sig-contributor-experience/annual-report-2023.md:110]:
```suggestion
- [x] Other contributing docs (e.g. in devel dir or contributor guide) reviewed for accuracy and updated if needed
```

Comment (Priyankasaggu11929) [reply sig-contributor-experience/annual-report-2023.md]: addressed in latest commit refresh.

**Comment by jberkus** [comment]
/lgtm
on behalf of Elections WG

File: sig-contributor-experience/annual-report-2023.md [hunk 6]
```diff
@@ -3,17 +3,65 @@
...
+        - started process to get access to kubernetes twitter account
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
+        - SIG spotlight blogs
+            - [Spotlight on SIG CLI](https://kubernetes.io/blog/2023/07/20/sig-cli-spotlight-2023/)
+            - [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Conformance](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Production Readiness](https://kubernetes.io/blog/2023/11/02/sig-architecture-production-readiness-spotlight-2023/)
+            - [Introducing SIG etcd](https://kubernetes.io/blog/2023/11/07/introducing-sig-etcd/)
+            - how many by new contributors?
+        - new subproject leadership roles?
```
Comment (Priyankasaggu11929) [inline sig-contributor-experience/annual-report-2023.md:50]: @kaslin, I've added a list of Spotlight blogs published in 2023. Please review/add/update.

Also the pending 2 questions:

 - how many (SIG spotlight blogs) by new contributors?
 - new subproject leadership roles in contributor-comms?

File: sig-contributor-experience/annual-report-2023.md [hunk 7]
```diff
@@ -3,17 +3,65 @@
...
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - got access to Linkedin
+        - started process to get access to kubernetes twitter account
```
Comment (humblec) [inline sig-contributor-experience/annual-report-2023.md:41]: nit: s/twitter/X account ?

Comment (Priyankasaggu11929) [reply sig-contributor-experience/annual-report-2023.md]: Thanks @humblec. Updated in recent commit refresh.

**Comment by Priyankasaggu11929** [comment]
@MadhavJivrajani @palnabarun @kaslin, ping for another review.

I've commented out any pending sections (let's fill them in separate PRs whenever we have the information handy).

@mrbobbytables, need your review/lgtm/approve. Thanks!

File: sig-contributor-experience/annual-report-2023.md [hunk 8]
```diff
@@ -3,17 +3,68 @@
...
+            - [China, hosted in Shanghai](https://www.kubernetes.dev/events/2023/kcscn/) (special thanks to @pacoxu for [leading the event](https://github.com/kubernetes/community/issues/7510))
+            - [North America, hosted in Chicago, Illinois](https://www.kubernetes.dev/events/2023/kcsna/)
+
+    - Mentoring:
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
```
Comment (MadhavJivrajani) [inline sig-contributor-experience/annual-report-2023.md:37]:
```suggestion
        - Facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
```

File: sig-contributor-experience/annual-report-2023.md [hunk 9]
```diff
@@ -3,17 +3,68 @@
...
+
+    - Contributor-comms:
+        - got access to [Kubernetes Linkedin account](https://www.linkedin.com/company/kubernetes/)
+        - started process to get access to Kubernetes X (earlier, Twitter) account – [@kubernetesio](https://x.com/kubernetesio)
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
```
Comment (MadhavJivrajani) [inline sig-contributor-experience/annual-report-2023.md:42]:
```suggestion
        - Got access to [Kubernetes Linkedin account](https://www.linkedin.com/company/kubernetes/)
        - Started process to get access to Kubernetes X (earlier, Twitter) account – [@kubernetesio](https://x.com/kubernetesio)
        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
```

File: sig-contributor-experience/annual-report-2023.md [hunk 10]
```diff
@@ -3,17 +3,68 @@
...
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - got access to [Kubernetes Linkedin account](https://www.linkedin.com/company/kubernetes/)
+        - started process to get access to Kubernetes X (earlier, Twitter) account – [@kubernetesio](https://x.com/kubernetesio)
```
Comment (MadhavJivrajani) [inline sig-contributor-experience/annual-report-2023.md:41]: I think this is complete now! We can write this as "Got access" 🎉

**Review by MadhavJivrajani — COMMENTED** [review]
LGTM for the most part, couple of nits and a minor comment, thanks a bunch for driving this @Priyankasaggu11929 ❤️

File: sig-contributor-experience/annual-report-2023.md (hunk 10 above)
Comment (Priyankasaggu11929) [reply sig-contributor-experience/annual-report-2023.md:41]: The last I remember from a discussion in slack, is we are still waiting for the email to change to some Kubernetes alias (with kubernetes/CNCF members)?

Just wanted to confirm if that's also done?

Comment (kaslin) [reply sig-contributor-experience/annual-report-2023.md]: "got access" is accurate. We have the credentials. They're in one password. We just don't have the guidelines in place for who gets access, and like you said, we do need to spin up a new email alias. But we have full access and the power to do whatever we need to with the account.

Comment (Priyankasaggu11929) [reply sig-contributor-experience/annual-report-2023.md]: changed to "got access". Thanks Madhav, Kaslin!

File: sig-contributor-experience/annual-report-2023.md [hunk 11]
```diff
@@ -3,17 +3,68 @@
...
+        - 5 SIG spotlight blogs (2/5 contributed by new contributors)
+            - [Spotlight on SIG CLI](https://kubernetes.io/blog/2023/07/20/sig-cli-spotlight-2023/)
+            - [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Conformance](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Production Readiness](https://kubernetes.io/blog/2023/11/02/sig-architecture-production-readiness-spotlight-2023/)
+            - [Introducing SIG etcd](https://kubernetes.io/blog/2023/11/07/introducing-sig-etcd/)
+        - New leadership roles in subproject:
+            - Onboarded 1 [Comms Tech Lead](https://github.com/kubernetes/community/commit/ba3d34a001abd3d50004f87ae44385efa5b2695d)
+            - Onboarded 1 [Social Media Coordinator shadow](https://github.com/kubernetes/community/pull/7438)
```
Comment (Priyankasaggu11929) [inline sig-contributor-experience/annual-report-2023.md:52]: @kaslin, I've added rest of the contributor-comms points (we discussed during yesterday's (June 5, 2024) leads sync call). Please review. Thanks!

**Review by mrbobbytables — COMMENTED** [review]
It won't let me comment directly on it - but line 96 where it has a new subproject theres a <no value> inserted that is breaking the link. It's a small thing, but after that I think it's g2g to be merged :)

**Comment by Priyankasaggu11929** [comment]
@mrbobbytables, thanks for pointing! :pray: 

Fixed the broken link now. :slightly_smiling_face:

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm

**Comment by mrbobbytables** [comment]
Thanks! 
/lgtm
/approve

**Pull request merged**